<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentalcar_</class>
 <widget class="QDialog" name="rentalcar_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>렌트카 정보 등록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stock_text">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 모델명</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>대여가</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>기름 종류</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>보유개수</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="type_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>휘발유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LPG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전기</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이름 중복확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_text">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 CC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
